extensions/azure-keyvault/runtime/src/main/java/io/quarkiverse/azure/keyvault/secret/runtime/KeyVaultSecretClientProducer.java

```diff
@@ -18,29 +19,29 @@ public class KeyVaultSecretClientProducer {
 
     @Produces
     public SecretClient createSecretClient() {
-        if (!secretConfiguration.enabled) {
+        if (!secretConfiguration.enabled()) {
```

Comment:
How about logging a message here?

Reply:
Hello @backwind1233 . This is also a good point. However, this is another case where a comprehensive approach to logging is needed. I suspect the Azure SDK has some guidelines on logging. I ask @galiacheng to create a User Story to review architectural guidance and make a recommendation, then undertake the work to instrument the codebase with logging according to this guidance. In the meantime, I don't want to take a piecemeal action.

Reply:
Thanks for the comments @backwind1233 @edburns, I've created a user story for logging in [quarkus-833](https://dev.azure.com/azure-redhat-projects/azure-redhat-projects/_workitems/edit/833).